test/rollup.config.js

```diff
@@ -0,0 +1,93 @@
+import easySauce from 'easy-sauce';
+
+import babel from 'rollup-plugin-babel';
+import browsersync from 'rollup-plugin-browsersync';
+import commonjs from 'rollup-plugin-commonjs';
+import { eslint } from 'rollup-plugin-eslint';
+import json from 'rollup-plugin-json';
+import multiEntry from 'rollup-plugin-multi-entry';
+import nodeResolve from 'rollup-plugin-node-resolve';
+import nodeGlobals from 'rollup-plugin-node-globals';
+
+const footer = 'this && this.Marionette && (this.Mn = this.Marionette);';
+
+const isSauce = process.env.NODE_ENV === 'sauce';
+
+function runSauce() {
+  easySauce({
+    name: 'Marionette.js',
+    username: process.env.SAUCE_USERNAME,
+    key: process.env.SAUCE_ACCESS_KEY,
+    port: '8080',
+    testPath: '/test/sauce.html',
+    framework: 'mocha',
+    platforms: [
```

Comment:
safari was removed from the matrix as, though the tests passed, the runner never registered with mocha, so something is up somewhere with safari and the sauce chain.  Not sure how to fix it, but I don't think safari should be a problem anyhow.